test/safientCore.js

```diff
@@ -0,0 +1,44 @@
+const { SafientSDK } = require('../dist/index');
+const { randomBytes } = require('crypto');
+const chai = require('chai');
+const expect = chai.expect;
+chai.use(require('chai-as-promised'));
+
+describe('SafientSDK', async () => {
+  it('Should register a new user', async () => {
+    try {
+      const seed = new Uint8Array(randomBytes(32));
+      const sc = new SafientSDK(seed);
+      const conn = await sc.safientCore.connectUser();
+
+      // SUCCESS : create user A
+      await sc.safientCore.registerNewUser(conn, 'DID:A', 'A', 'A@test.com', 0);
```

Comment:
Need to delete the user after the test as this will throw an error the next time. Add a `beforeEach` to clean up the users.

Reply:
Yeah came across this problem. Wasn't sure how to go about it. I will use `beforeEach` to clean up 👍